README.md

```diff
@@ -238,6 +272,17 @@ export default defineConfig({
 })
 ```
 
+```js
+export default {
+  title: 'Main Video',
+  name: 'mainVideo',
+  type: 'mux.video',
+  options: {
+    encoding_tier: 'baseline',
```

Comment:
`encoding_tier` has been renamed to `video_quality` (and `smart` -> `plus` and `baseline` -> `basic`)

I don't think the Sanity plugin supports `video_quality` yet, so this isn't an actionable comment. Just making a note that, at some point, we'll need to update the Sanity plugin to reflect this.